webpack.config.js

```diff
@@ -18,14 +18,6 @@ module.exports = {
   module: {
     loaders: [
       {
-        test: /\.js$/,
-        // exclude: /(node_modules)/,
-        loader: 'babel',
```

Comment:
I removed it from the webpack config because it's made redundant -- the webpack entry point now points to `lib/`, which is already pre-transpiled.